deployments/warp_routes/AMPHRETH/ethereum-zircuit-config.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
+  - addressOrDenom: "0xA4e45cB601d43818518FB6023b8070d2564b935f"
```

Comment:
Not mandatory but you probably want a `logoURI` field in these token entries if they're gonna be used in a UI

Reply:
we don't have one for now, will just default to whatever the UI uses